src/script/common/c_content.cpp

```diff
@@ -1513,6 +1553,49 @@ ToolCapabilities read_tool_capabilities(
 	return toolcap;
 }
 
+/******************************************************************************/
+WearBarParams read_wear_bar_params(
+		lua_State *L, int table)
+{
+	WearBarParams params;
+	std::basic_string<char> default_color_string;
+	if (getstringfield(L, table, "default", default_color_string))
+		parseColorString(default_color_string, params.defaultColor, false);
+	bool blend;
+	if (!getboolfield(L, table, "blend", blend))
+		blend = false;
+	params.blend = blend;
+
+	int table_values = lua_gettop(L);
+	lua_pushnil(L);
+	while(lua_next(L, table_values) != 0) {
+		// key at index -2 and value at index -1=
+		// values are stored in a flat table along with the default color,
+		// but values can be distinguished by them being a table
+		// (in blend mode, they can be distinguished by having a numerical key)
```

Comment:
What does `index -1=` mean? Did you mean `index -1 means that`?

Reply:
cleaned up the comment to this:

```c
		// key at index -2 and value at index -1 within table_values
		// color value entries are stored in a flat 'list' along with the default color,
		// but in non-blend mode color entries can be distinguished by that value being a table
		// (in blend mode, color entries can be distinguished by having a numerical key)
```